meson_options.txt

```diff
@@ -14,4 +14,6 @@ option('persist-mac', type: 'boolean',
        description: 'Permit the MAC address to be written to the systemd.network config')
 option('force-sync-mac', type: 'boolean',
        description: 'Force sync mac address no matter is first boot or not')
+option('default-dhcp6-without-ra', type: 'boolean',
+       description: 'Enables stateful DHCPv6 by default')
```

Comment:
This value is an enumeration. true/false will not work

Reply:
> This value is an enumeration. true/false will not work

Update code, please review.